cmd/kubectl-gadget/version.go

```diff
@@ -36,6 +40,46 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Show version",
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(version)
+		fmt.Println("Client version:", version)
+
+		client, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
+		if err != nil {
+			fmt.Println("Error while creating a new clientset:", err)
+			return
+		}
+
+		opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
+		pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
+		if err != nil {
+			fmt.Println("Error while getting the gadget pods:", err)
+			return
+		}
+
+		serverVersions := make(map[string]struct{})
+		for _, pod := range pods.Items {
+			image := pod.Spec.Containers[0].Image
+
+			// Get the image tag
+			parts := strings.Split(image, ":")
+			if len(parts) != 2 {
+				continue
+			}
+
+			versionStr := parts[1]
+			if _, ok := serverVersions[versionStr]; !ok {
+				serverVersions[versionStr] = struct{}{}
+			}
+		}
+
+		if len(serverVersions) == 0 {
+			fmt.Println("Server version:", "not installed")
+		} else {
+			if len(serverVersions) > 1 {
+				fmt.Println("Warning: Multiple deployed versions detected")
```

Comment:
What about printing the node, pod and version?

Reply:
When do we print the node, pod and version?
Only if they are different?

What about if 50 pods are version X and 50 pods are version Y?
Do we print all of them?

These are the kind of questions that i would have

Reply:
Good point. Forget about my proposal 🤣.

Reply:
Yes, that is intended to enumerate all detected versions